### PR TITLE
Switchstix: Fix issue with eating 10,000 pieces, refactor

### PR DESCRIPTION
**Pull request by rude-jerk**
**_I affirm:_**
- [x] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [x] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/zones/Castle_Zvahl_Baileys/npcs/Switchstix.lua [hunk 1]
```diff
@@ -9,291 +9,303 @@ require("scripts/globals/keyitems")
...
+    SHELL_10000 = 1451,
 }
 
-function hasRelic(entity, checktype)
-    -- Type 1 == Player is triggering NPC
-    -- Type 2 == Player is trading an item
+local relics =
+{
+    -- Spharai
+    [18260] = { { 1460, 1459, 665 }, currency.BYNE_100, 4, 1, 1 }, -- Relic Knuckles
```
Comment (TeoTwawki) [inline scripts/zones/Castle_Zvahl_Baileys/npcs/Switchstix.lua:31]: keys! the new table has keeeeeys!


Listen.. I love you. no homo.

Comment (rude-jerk) [reply scripts/zones/Castle_Zvahl_Baileys/npcs/Switchstix.lua:31]: You should have seen me almost physically struggle with not creating _another_ new table to enumerate the relic weapon stage IDs into item names. 🙀

**Comment by zach2good** [comment]
That's some handsome looking code right there! Ping me when you're done pushing changes and I'll give it a final once over and a green stamp 👍

File: .gitignore [hunk 2]
```diff
@@ -63,6 +63,9 @@ ipch/
 # ReSharper is a .NET coding add-in
 _ReSharper*
 
+# .idea is created by CLion
```
Comment (zach2good) [inline .gitignore:66]: Yes, a fine choice 🦁

**Review by zach2good — APPROVED** [review]
Looks good! Like I say, ping me when you want me to hit the button

**Comment by rude-jerk** [comment]
Okay @zach2good I'm done fiddling! Good to go.

**Pull request merged**